package.json

```diff
@@ -60,6 +60,8 @@
     "react-dom": "^17.0.2",
     "react-ga": "^3.1.2",
     "react-gtm-module": "^2.0.11",
+    "react-helmet": "^6.1.0",
```

Comment:
Can you explain why these 2 new dependencies are needed in this PR?

Reply:
Another oversight would have happened while I was reverting some commits. I've removed them. Thanks so much @akshatnema